contracts/pcv/curve/StableSwapOperatorV1.sol

```diff
@@ -285,7 +285,35 @@ contract StableSwapOperatorV1 is PCVDeposit {
         return _daiOut;
     }
 
+    /// @notice returns the token address in which this contract reports its
+    /// balance.
+    /// @return the DAI address
     function balanceReportedIn() public view override returns(address) {
         return _dai;
     }
+
+    /// @notice gets the resistant token balance and protocol owned fei of this deposit
+    /// for balance, returns half of the theoretical USD value of the LP tokens, even though
+    /// there may be some slippage when withdrawing to DAI only. for fei, take the
+    /// same value (assumes there is no broken peg in the pool).
+    /// @return resistantBalance the resistant balance of DAI (theoretical USD value)
+    /// @return resistantFei the resistant balance of FEI (theoretical USD value)
+    function resistantBalanceAndFei() public view override returns (
```

Comment:
the constants in here should be declare globally and the ratio thresholds should be settable by governance